src/replicache.ts

```diff
@@ -1156,6 +1163,7 @@ export class Replicache<MD extends MutatorDefs = {}> {
           name,
           mutatorImpl,
           args,
+          performance.now(),
```

Comment:
should this be `this.now()`?

Reply:
No, I thought at first that `Now` would have to be injected for testing, but you are already controlling the timers in the tests with monkey patching.